### PR TITLE
code works successfully

### PR DESCRIPTION
**Pull request by noor848**

### CODE REVIEW

**Comment by noor848** [comment]
https://leetcode.com/problems/linked-list-cycle-ii/

File: Task1.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    public int val;
+    public ListNode next;
+    public ListNode(int x) {
+        val = x;
+        next = null;
+   }
+}
+
+public class Task1
+{  public static ListNode DetectCycle(ListNode head)
```
Comment (hayatzsala) [inline Task1.cs]: CLEAN CODE: keep brackets alone in the line

File: Task1.cs [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            }
+
+            NodeStore.Add(listOne, listOne.next);
+            listOne = listOne.next;
+        }
+        return null;
+    }
+
+    public static void Main(string[] args)
+    {
```
Comment (hayatzsala) [inline Task1.cs]: @noor848 extract Main to Program class please same as Stack task

File: Task1.cs [hunk 3]
```diff
@@ -0,0 +1,67 @@
+﻿
```
Comment (hayatzsala) [inline Task1.cs:1]: @noor848  Could you re-solve this problem without using Dictionary? think on that, Don't update the PR, will discussed it.

**Review by hayatzsala — APPROVED** [review]

**Pull request merged**